dev/common/check_pulp_template.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+
+# awcrosby 2022-02-25 11:15AM
+#   everytime we update pulp_ansible or pulpcore we run that plugin template 
+#   https://github.com/ansible/galaxy_ng/wiki/Updating-Dependencies, 
+#   its worth checking if [any modified files] gets overwritten
+
+set -e
+
+CHECKOUT=$(pwd)
+BASEDIR=$(python -c 'import tempfile; print(tempfile.mkdtemp(prefix="pulp_template_check-"))')
+PLUGIN_GIT_REF=$(cat .github/template_gitref | awk -F\- '{print $NF}' | cut -dg -f2)
+
+rm -rf $BASEDIR; mkdir -p $BASEDIR
+cp -Rp $CHECKOUT $BASEDIR/galaxy_ng
+git clone https://github.com/pulp/plugin_template $BASEDIR/plugin_template
+
+cd $BASEDIR/plugin_template
+git checkout $PLUGIN_GIT_REF
+./plugin-template --github galaxy_ng
+
+echo "Results ..."
+cd $BASEDIR/galaxy_ng
+MODIFIED_FILES=$(git status 2>/dev/null | grep 'modified:' | awk '{print $2}')
+EXIT_CODE=0
+for MF in $MODIFIED_FILES; do
+    echo "FAILURE template-plugin would modifiy $BASEDIR/galaxy_ng/$MF"
```

Comment:
So this will also fail when we run plugin_template to explicitly modify the files? how is this script differentiating when we really want to have the files edited by the template engine from when we do that by mistake?